ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/ipv6-neighbors.j2

```diff
@@ -0,0 +1,34 @@
+{#
+ Copyright (c) 2023-2024 Arista Networks, Inc.
+ Use of this source code is governed by the Apache License 2.0
+ that can be found in the LICENSE file.
+#}
+{# doc - ipv6 static neighbors #}
+{% if ipv6_neighbor is arista.avd.defined %}
+
+## IPv6 Neighbors
```

Comment:
we should start with ### level as other templates in routing.j2 start at the same level.

Reply:
fixed